modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -65,12 +84,22 @@ public class LeaseTracker {
      *
      * @param vaultManager Vault manager.
      * @param msManager Meta storage manager.
+     * @param longLeaseInterval Long lease interval in {@code TimeUnit.MILLISECONDS}.  The interval is used between lease granting attempts.
+     * @param busyLock Busy lock to protect {@link PlacementDriver} methods on corresponding manager stop.
      */
-    public LeaseTracker(VaultManager vaultManager, MetaStorageManager msManager) {
+    public LeaseTracker(
+            VaultManager vaultManager,
+            MetaStorageManager msManager,
+            long longLeaseInterval,
+            IgniteSpinBusyLock busyLock
+    ) {
         this.vaultManager = vaultManager;
         this.msManager = msManager;
+        this.longLeaseInterval = longLeaseInterval;
+        this.busyLock = busyLock;
```

Comment:
It is forbidden to take the lock object from where it was created.

Reply:
Fixed.